packages/nextjs/app/builders/0xE451141fCE63EB38e85F08a991fC5878Ee6335b2/page.tsx

```diff
@@ -0,0 +1,181 @@
+              <ul className="mt-2 space-y-2 text-sm text-center">
+                <li>
+                  <a
+                    className="text-cyan-500 dark:text-cyan-400 hover:text-cyan-700 dark:hover:text-cyan-200 hover:underline"
+                    href="https://github.com/Parizval"
+                    target="_blank"
+                    rel="noreferrer"
+                  >
+                    GitHub Profile
+                  </a>
+                </li>
+                <li>
+                  <a
+                    className="text-cyan-500 dark:text-cyan-400 hover:text-cyan-700 dark:hover:text-cyan-200 hover:underline"
+                    href="https://x.com/Parizval_Wizard"
+                    target="_blank"
+                    rel="noreferrer"
+                  >
+                    X/Twitter
+                  </a>
+                </li>
+                <li>
+                  <Link
+                    className="text-cyan-500 dark:text-cyan-400 hover:text-cyan-700 dark:hover:text-cyan-200 hover:underline"
+                    href="/"
+                  >
+                    Batch Home
+                  </Link>
+                </li>
+                <li>
+                  <Link
+                    className="text-cyan-500 dark:text-cyan-400 hover:text-cyan-700 dark:hover:text-cyan-200 hover:underline"
+                    href="/debug"
+                  >
+                    Debug Tools
+                  </Link>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </section>
+
+        {/* Right Column */}
+        <section className="md:col-span-2">
+          <div className="rounded-2xl border border-cyan-500/30 shadow-lg shadow-cyan-500/10 p-6 bg-gray-100 dark:bg-gray-900/50 backdrop-blur-sm">
+            <h2 className="text-xl font-bold text-cyan-600 dark:text-cyan-300 mb-4">Web3 Projects Portfolio</h2>
+            <div className="space-y-4">
+              <div className="rounded-lg border border-cyan-500/20 p-4 bg-white dark:bg-gray-800/40 hover:bg-gray-200 dark:hover:bg-gray-800/70 transition-colors">
+                <h3 className="font-semibold text-cyan-500 dark:text-cyan-400">Stylish Lending Hook</h3>
+                <p className="text-sm text-gray-600 dark:text-gray-400 mt-1">
+                  🪝 A CowSwap Hook which deposits the output Tokens into Lending Protocols.
+                </p>
+                <a
+                  href="https://ethglobal.com/showcase/stylish-lending-hook-t1c3u"
+                  className="text-xs text-cyan-600 dark:text-cyan-500 hover:underline mt-2 inline-block"
+                  target="_blank"
+                  rel="noreferrer"
+                >
+                  View Repo
+                </a>
+              </div>
+              <div className="rounded-lg border border-cyan-500/20 p-4 bg-white dark:bg-gray-800/40 hover:bg-gray-200 dark:hover:bg-gray-800/70 transition-colors">
```

Comment:
You should only have hover effects for interactive elements. This hover effect misleads the user into thinking they can click it to navigate.